tests/unit/asio_service_test.cxx

```diff
@@ -639,13 +639,33 @@ int response_hint_test(bool with_meta) {
     CHK_OK( s2.getTestSm()->isSame( *s1.getTestSm() ) );
     CHK_OK( s3.getTestSm()->isSame( *s1.getTestSm() ) );
 
+    _msg("enable batch size hint with negative value\n");
+    for (RaftAsioPkg* ee: pkgs) {
+        ee->getTestSm()->set_next_batch_size_hint_in_bytes(-1);
+    }
+
+    // on Mac, due to sync append, leader reelect will happen
+    // on Ubuntu, due to sync append, 100 times append are very slow
```

Comment:
Please adjust `client_req_timeout_` to a shorter number for this test, so as to make it faster, using `update_params` API. The current default is 10 seconds. 
https://github.com/eBay/NuRaft/blob/6d371ddcc6bc0a9b8aaad4e57e9859adbc111d85/tests/unit/raft_package_asio.hxx#L123

Reply:
Also may need to adjust `leadership_expiry_` to `-1`, to avoid unwanted leader election.

Reply:
update `client_req_timeout_` is done.
About adjusting `leadership_expiry_ `, it may be not useful. On Mac, the re-elect is caused by the follower did not receive the heartbeat and is triggered by follower, not by leader.

Besides, with negative hint size, raft only can not send append entry request with raft logs, but the empty append entry request still can work as a heartbeat.

Since we only do client request on raft server 1, so if leader changes, then the following client request on old leader may fail quickly. For this situation, the purpose of this test case may can not be achieved, but it should not influence the successful finish of this test case.